components/features/task-creation/task-create-modal.tsx

```
@@ -0,0 +1,455 @@
+									Add
+								</Button>
+							</div>
+							{taskData.labels.length > 0 && (
+								<div className="flex flex-wrap gap-2 mt-2">
+									{taskData.labels.map((label) => (
+										<Badge key={label} variant="secondary" className="flex items-center space-x-1">
+											<span>{label}</span>
+											<button
+												onClick={() => handleRemoveLabel(label)}
+												className="ml-1 hover:text-red-600"
+											>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)